x-pack/plugin/esql/qa/testFixtures/src/main/resources/docs-IT_tests_only.csv-spec

```diff
@@ -1,3 +1,15 @@
+// This file contains any ESQL snippets from the docs that don't have a home
+// anywhere else. The Isle of Misfit Toys. When you need to add new examples
+// you should try to convert an existing test in one of it's file. Just add
+// the comments in whatever file it is in. If you have to write a new test
+// for the docs put it in whatever file matches it's "theme" best.
```

Comment:
`it's` -> `its` ?

Reply:
++.

I have been making this mistake since I learned to write and I'll probably continue making this mistake for as long as I can write.